elements/nuxeo-document-comments/nuxeo-document-comment-thread.html

```diff
@@ -352,6 +357,9 @@
       },
 
       /** Visibility Methods **/
+      _allowReplies: function (level) {
```

Comment:
white space between function and parmeters:
```suggestion
      _allowReplies: function(level) {
```